action.yml

```diff
@@ -1,35 +1,59 @@
-name: Terraform Auto-Approve
-description: 'Auto-approve terrafrom changes that do not have state changes'
+name: Terraform auto-merge
+description: 'Auto-approve or Auto-merge terrafrom changes that do not have state changes'
 inputs:
   github-token:
     description: 'Github token of the bot used for the auto-approving'
     required: true
   terraform-directory:
-    description: 'Root directory for the terraform files'
+    description: 'Root directory for the terraform files e.g. terraform/'
     required: true
+  merge:
+    description: 'Enable auto-merging'
+    required: false
+    default: false
+  force-merge:
+    description: 'Enable auto-merging using admin access'
+    required: false
+    default: false
 runs:
   using: 'composite'
   steps:
+  - if: github.event_name != 'pull_request'
+    run: exit 1
```

Comment:
Wouldn't `exit 0` make more sense? No need to mark the workflow as failed, right?

Reply:
I think failing is better, you shouldn't even use the action in a non-pr workflow

Reply:
Can imagine it being used on both PR and push events for example. But, we'll see if this'll cause annoyances or not :wink: